### ISSUE TITLE
Pro-tip: Workaround for error when trying to add bit remote

### ISSUE THREAD

**Issue by gscottolson** [teambit/bit#682]
I followed the instructions for setting up bit on a server. Upon adding the bit remote, I get the following error:

```
bit remote add ssh://[ ip redacted ]:/opt/bit/first-scope
fatal: remote scope "Error" not found.
```

The set up instructions asked the user to create a `bit` account on the bit server. So I added the following to the url after `ssh://`: **`bit@`**

```
bit remote add ssh://bit@[ ip redacted ]:/opt/bit/first-scope
added remote scope 'first-scope' with host 'ssh://bit@[ ip redacted ]:/opt/bit/first-scope'
```

If this seems useful, I am happy to create a PR to add this to the docs. 🍻

**Comment by itaymendel** [comment]
hi @gscottolson ! thanks for posting this issue :)

currently, the docs is still a private project (which we plan to open-source soon), so you can't add your own fix right now. However, with your permission, i can do a quick fix to the docs, and add the ssh-user to the command (and of course, give you credit for it).

what do you think?

**Comment by gscottolson** [comment]
Of course. And thanks for your work on this project.

**Comment by itaymendel** [comment]
thanks! i'll publish a new version of the docs shortly.

we're trying to solve a problem that we encounter daily, and hope the solution can fit other developers :)

**Comment by lamtranweb** [comment]
I followed instructions on the docs, but the following command to my own server:

```bash
bit remote add ssh://[bit-username]@[bit-server]:/opt/bit/[scope-name]
```
Still asks for `username` and `password` even though I already set up SSH authentication (as I'm able to ssh into my server without providing username/password).

I went ahead and added a password to my user anyway.  But providing the password, or no matter what I do, still end up with:

`authentication failed. see troubleshooting at https://docs.bitsrc.io/docs/authentication-issues.html`
